docker/trustlines/config.toml

```diff
@@ -0,0 +1,35 @@
+[relay]
```

Comment:
I don't care too much, but why did the configuration got that much extended in comparison the previous version? Are this defaults or they related to the new setup by the documentation to provide the user with a quick solution to go for?

Reply:
Well, most of them are new options.